src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__cell calendar__cell--1"></div>
```

Comment:
Remove numbers from the classes, you can use `nth-child`

Reply:
done
Thank you for review